source/_patterns/04-pages/msa.mustache

```diff
@@ -24,21 +24,25 @@
       </div>
 
 
-    <div class="wrapper">
+    <div class="wrapper wrapper--listing wrapper--listing-selector">
 
       <div class="grid">
 
         <div class="listing-column-container">
 
-          <div class="grid__item large--eight-twelfths listing-column-main">
+          {{#viewSelector}}
+            {{> molecules-view-selector }}
+          {{/viewSelector}}
+
+          <div class="grid__item large--eight-twelfths listing-column-main primary-column" id="primaryListing">
 
             {{#mainTeasers}}
               {{> organisms-listing-teasers}}
             {{/mainTeasers}}
 
           </div>
 
-          <div class="grid__item large--four-twelfths listing-column-secondary">
+          <div class="grid__item large--four-twelfths listing-column-secondary secondary-column" id="secondaryListing">
```

Comment:
Here too, the "one-whole" class is missing.

Reply:
Done, also "large--eight-twelfths" is changed to "x-large--eight-twelfths"